src/components/views/settings/tabs/user/LabsUserSettingsTab.js

```diff
@@ -35,7 +35,8 @@ export class LabsSettingToggle extends React.Component {
     render() {
         const label = SettingsStore.getDisplayName(this.props.featureId);
         const value = SettingsStore.getValue(this.props.featureId);
-        return <LabelledToggleSwitch value={value} label={label} onChange={this._onChange} />;
+        const canChange = SettingsStore.canSetValue(this.props.featureId, null, SettingLevel.DEVICE);
```

Comment:
Ah interesting, I guess I hadn't fully realised from the proposal that we would show all options, but disable the forced ones (instead of hiding them or something else). I guess that seems fine though.

Reply:
it does feel a bit weird - the theory is that users get to play with labs options more readily. Something to retro on for the next iteration.